src/Poseidon/CLI/Serve.hs

```diff
@@ -102,30 +101,87 @@ instance FromJSON ArchiveSpec where
         <*> v .:? "URL"
         <*> v .:? "dataURL"
         <*> ((v .:? "excludeFromMap") >>= maybe (return []) return)
+        <*> parseCLIretiredPackages v
```

Comment:
Noticed when testing: With this setup the `retiredPackages` field is mandatory in the .yml file, just like `name` and `paths`. I think it would be better to have it optional like the other fields.

Reply:
Great catch, indeed! Just fixed it.